mmdeploy/backend/tensorrt/utils.py

```diff
@@ -118,6 +118,17 @@ def from_onnx(onnx_model: Union[str, onnx.ModelProto],
 
     config = builder.create_builder_config()
     config.max_workspace_size = max_workspace_size
+    try:
+        import torch
```

Comment:
I am not sure if it is a good idea to get CUDA version from torch.
This API is expected to be able to use without PyTorch. Do we have other ways to check the CUDA version?

Reply:
I have no better idea

Reply:
Some one use mmdeploy  -->  guy has installed mmxxx repo ---> pytorch already installed.

Reply:
Users might export ONNX on host A and convert to TensorRT on host B (without PyTorch). And the PyTorch has it's builtin CUDA which might be different from the one in the system.
Can we use `ctypes` to load runtime api for the version check?

```python
from ctypes import cdll, pointer, c_int32
dll = cdll.LoadLibrary('libcudart.so')
version = c_int32(0)
dll.cudaRuntimeGetVersion(pointer(version))
print(version)
```

Reply:
> I have no better idea. TRT input comes from torch, and torch depends on cuda... I think import torch is a must.

I didn't get `TRT input comes from torch`

Reply:
Have you ever deploy model with mmdeploy without  torch and `mmcls` ?